content/efcore/getting-started/querying.md

```diff
@@ -18,15 +18,15 @@ using (var context = new LibraryContext())
 :::repl{data-name=loading-all-entities}
 :::
  
-In order to interact with the database via EF Core, we must first create an instance of our context (`LibraryContext`). `LibraryContext` extends EF Core's [`DbContext`](https://msdn.microsoft.com/library/system.data.entity.dbcontext) class which manages entity objects during runtime, and allows us to perform common database operations like running queries.
+In order to interact with the database via EF Core, we must first create an instance of our context. Our context, `LibraryContext`, extends EF Core's [`DbContext`](https://msdn.microsoft.com/library/system.data.entity.dbcontext) class which manages entity objects during runtime, and allows us to perform common database operations like running queries. We made `LibraryContext` in an earlier lesson, ["Understanding Data Models"](understanding-data-models.md).
```

Comment:
Because we explain `LibraryContext` and its relation to `DbContext` in a previous lesson, lets remove all of this. Combine the first sentence with the next paragraph.